File: WikipediaDeathsPages.Service/Dtos/ExistingEntryDto.cs
using System;

namespace WikipediaDeathsPages.Service.Dtos
{
    public class ExistingEntryDto
    {
        public string ArticleName { get; set; }
        public string ArticleLinkedName { get; set; }
        public string Information { get; set; }
        public string Reference { get; set; }
        public string ReferenceUrl { get; set; }
        public DateTime DateOfDeath { get; set; }
        public string Uri { get; set; }
        public string WikiText { get; set; }
        public int NotabilityScore { get; set; }
        public bool Keep { get; set; }
        public string ReasonKeepReject { get; set; }
    }
}

File: WikipediaDeathsPages.Web/ClientApp/src/app/dto/ExistingEntryDto.ts
export interface ExistingEntryDto{
    dateOfDeath: Date;
    articleName: string;
    information: string;
    reference: string;
    referenceUrl: string;
    uri: string;
    wikiText: string;
    notabilityScore: number;
    keep: boolean;
    reasonKeepReject: string;
}